apps/designer/app/designer/features/topbar/menu/menu.tsx

```diff
@@ -37,6 +38,9 @@ const menuItemCss = {
 };
 
 const ThemeMenuItem = () => {
+  if (isFeatureEnabled("dark") === false) {
+    return null;
```

Comment:
null?

Reply:
we will get rid of these once react supports undefined, they had an RFC for this and maybe even already in v18